curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/66bbb13f90639e7a6d664153.md

```diff
@@ -0,0 +1,167 @@
+---
+id: 66bbb13f90639e7a6d664153
+title: Step 34
+challengeType: 0
+dashedName: step-34
+---
+
+# --description--
+
+While `getMode` is looking great, the `forEach` loop can simplified further. In order to do so, the function will be turned into an `arrow function expression`. This kind of expression is known for not using curly braces and shorter syntax. 
+
+An example of such an expression is the following: 
+
+```js
+const numbersArr = [1, 2, 3, 4, 5, 1, 2, 3, 4, 5, 1, 2, 3, 4];
+const counts = {};
+numbersArr.forEach(el => counts[el] = (counts[el] || 0) + 1)
```

Comment:
Not sure how I feel about an example that could pretty much be copy-pasted.

Reply:
Oh. Yeah that's an issue. :( I should probably fix it.

Reply:
I've updated the example.